symfony/client/src/orangehrmLeavePlugin/pages/configure/holiday/Holiday.vue

```diff
@@ -0,0 +1,271 @@
+        @delete="onClickDeleteSelected"
+      ></table-header>
+      <div class="orangehrm-container">
+        <oxd-card-table
+          :headers="headers"
+          :items="items?.data"
+          :selectable="true"
+          :clickable="false"
+          v-model:selected="checkedItems"
+          :loading="isLoading"
+          rowDecorator="oxd-table-decorator-card"
+        />
+      </div>
+      <div class="orangehrm-bottom-container">
+        <oxd-pagination
+          v-if="showPaginator"
+          :length="pages"
+          v-model:current="currentPage"
+        />
+      </div>
+    </div>
+    <delete-confirmation ref="deleteDialog"></delete-confirmation>
+  </div>
+</template>
+
+<script>
+import {computed, ref} from 'vue';
+import DeleteConfirmationDialog from '@orangehrm/components/dialogs/DeleteConfirmationDialog';
+import usePaginate from '@orangehrm/core/util/composable/usePaginate';
+import {navigate} from '@orangehrm/core/util/helper/navigation';
+import {APIService} from '@/core/util/services/api.service';
+import useSort from '@orangehrm/core/util/composable/useSort';
+
+const dataNormalizer = data => {
+  return data.map(item => {
+    return {
+      id: item.id,
+      name: item.name,
+      date: item.date,
+      recurring: item.recurring ? 'Yes' : 'No',
+      length: item.lengthName,
+    };
+  });
+};
+
+const defaultFilters = {
+  fromDate: null,
+  toDate: null,
+};
+
+export default {
+  components: {
+    'delete-confirmation': DeleteConfirmationDialog,
+  },
+  props: {
+    leavePeriod: {
+      type: Array,
+      required: true,
+    },
+  },
+
+  data() {
+    return {
+      headers: [
+        {name: 'name', slot: 'title', title: 'Name', style: {flex: 2}},
+        {name: 'date', title: 'Date', style: {flex: 2}},
+        {name: 'length', title: 'Full Day/ Half Day', style: {flex: 2}},
+        {name: 'recurring', title: 'Repeats Annually', style: {flex: 2}},
+        {
+          name: 'actions',
+          title: 'Actions',
+          slot: 'action',
+          style: {flex: 1},
+          cellType: 'oxd-table-cell-actions',
+          cellConfig: {
+            delete: {
+              onClick: this.onClickDelete,
+              component: 'oxd-icon-button',
+              props: {
+                name: 'trash',
+              },
+            },
+            edit: {
+              onClick: this.onClickEdit,
+              props: {
+                name: 'pencil-fill',
+              },
+            },
+          },
+        },
+      ],
+      checkedItems: [],
+    };
+  },
+
+  setup(props) {
+    const filters = ref({...defaultFilters});
+    // eslint-disable-next-line vue/no-setup-props-destructure
+    filters.value.fromDate = props.leavePeriod.startDate;
+    // eslint-disable-next-line vue/no-setup-props-destructure
+    filters.value.toDate = props.leavePeriod.endDate;
+    const {sortDefinition, sortField, sortOrder, onSort} = useSort({
+      sortDefinition: {},
```

Comment:
remove `sort` references if no `sortDefinition` is here. from the mockup, it seems this table does not support sort. so it is not needed.